Source/Charts/Renderers/BarChartRenderer.swift

```diff
@@ -18,6 +18,28 @@ import CoreGraphics
 
 open class BarChartRenderer: BarLineScatterCandleBubbleRenderer
```

Comment:
In the demo project the "Another Bar Chart" demo has a frame that is too large around the bars.

Reply:
I see it! Will fix tomorrow.